Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -100,9 +112,92 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         }
     }
 
+    func updateNumberOfSyncedDevices() {
+        var numberOfDevices = 0
+        syncedTabsHeaderCell.descriptionLabel?.hidden = true
+        syncedTabsHeaderCell.descriptionLabel?.removeFromSuperview()
```

Comment:
No need to remove and re-add the `UILabel`. It should be good enough to make it hidden if you don't need it. Also helps performance if you're not modifying subviews like this.